independent-publisher-connectors/IBM Watson Assistant/apiDefinition.swagger.json

```diff
@@ -0,0 +1,332 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "IBM Watson Assistant",
+    "description": "The IBM Watson Assistant service combines machine learning, natural language understanding, and an integrated dialog editor to create conversation flows between your apps and your users.",
+    "version": "1.0",
+    "contact": {
+      "name": "Lucas Titus",
+      "url": "https://uk.linkedin.com/in/lucas-titus-a59541176",
+      "email": "lucas.titus.21@ucl.ac.uk"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://cloud.ibm.com/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.ibm.com/us-en/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "AI"
+    }
+  ],
+  "host": "assistant.watson.cloud.ibm.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/sessions": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "session_id": {
+                  "type": "string",
+                  "description": "session_id"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Create a session",
+        "description": "Create a new session. A session is used to send user input to a skill and receive responses.",
+        "operationId": "CreateSession",
+        "parameters": [
+          {
+            "name": "version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "2021-11-27",
+            "description": "date of release of Watson Assistant version",
+            "x-ms-summary": "version"
+          }
+        ]
+      }
+    },
+    "/sessions/{session}": {
+      "delete": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
```

Comment:
@lucastitus empty schema is not allowed, please remove it or define a proper schema.